doc/api/tls.md

```diff
@@ -1213,6 +1213,30 @@ It may be useful for debugging.
 
 See [Session Resumption][] for more information.
 
+### `tlsSocket.getPeerX509Certificate()`
```

Comment:
This should be before `tlsSocket.getTLSTicket()` to preserve alphabetical ordering. (Although it appears `tlsSocket.exportExportKeyingMaterial` is in the wrong place in that list.)